ansible_collections/arista/avd/plugins/test/contains.py

```diff
@@ -5,9 +5,9 @@
 # arista.avd.contains
 #
 # Example:
-# A is [1, 2]
-# B is [3, 4]
-# C is [2, 3]
+# A >>> [1, 2]
```

Comment:
Hmmm - is it more comprehensible?
```suggestion
A = [1, 2]
```

Reply:
It looks like code then which ruff complains about.

Reply:
changed to a docstring